src/test/java/edu/carroll/cs389application/service/ImageServiceImplTest.java

```diff
@@ -0,0 +1,287 @@
+package edu.carroll.cs389application.service;
+
+import edu.carroll.cs389application.jpa.model.Login;
+import edu.carroll.cs389application.jpa.model.UserImage;
+import edu.carroll.cs389application.service.ImageServiceImpl.ErrorCode;
+import edu.carroll.cs389application.web.form.ImageForm;
+import edu.carroll.cs389application.web.form.LoginForm;
+import jakarta.validation.constraints.AssertTrue;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.util.Pair;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+
+import static org.junit.Assert.*;
```

Comment:
Wildcard imports are rarely used, so either user them consistently, or don't use them at all.  (Mixing and matching the use of wildcard imports is bad style.)